tools/st2_deploy.sh

```diff
@@ -515,6 +515,13 @@ install_st2client() {
   fi
   popd
 
+  # Write ST2_BASE_URL to env
+  if [[ "$TYPE" == "rpms" ]]; then
```

Comment:
Hm, a bit confused about this part - do we still need it?

CLI config should also work on Fedora just fine (but yeah, unlike global bashrc we are using here, CLI config is specific to a system user).